app/javascript/packages/address-search/components/post-office-no-results.tsx

```diff
@@ -0,0 +1,23 @@
+import { getAssetPath } from '@18f/identity-assets';
+import { t } from '@18f/identity-i18n';
+
+function PostOfficeNoResults({ address }) {
```

Comment:
Since this will be exposed in the interface for the AddressSearch component and the [other props refer to Locations instead of Results](https://github.com/18F/identity-idp/pull/9144/files#diff-5a100588d3dc31741fcaa498749d909f1b34707055b472fe9f851724f9f78802L8-L14), what about calling this something like `NoInPersonLocations`, `NoInPersonLocationsDisplay`, `NoInPersonLocationsFound`?

This is rendered inside the `InPersonLocations` component so I'm also inclined to keep the `InPerson` prefix on the component but the prop exposed in `AddressSearch` would be better as something like `noLocations` to match the other props

Reply:
Good idea to keep the name more consistent with our other component.  I used `NoInPersonLocationsDisplay`. resolved with commit 0656a611f8087cdcbc0b43473e76a3fd468a8708